### DISCUSSION TITLE
Usage with Markdown only?

### DISCUSSION THREAD

**Discussion by joelzwarrington** [KonnorRogers/rhino-editor#193] [Q&A]
In the project README:
> Currently this package does so using [TipTap](https://tiptap.dev/) but will most likely include another integration for [ProseMirror](https://prosemirror.net/) to allow for both Markdown + WYSIWYG editing.

Has the integration with ProseMirror for Markdown only been setup?

**Comment by KonnorRogers** [comment]
@joelzwarrington nothing has been setup yet, I've been sitting on my hands on this hoping TipTap supports this out of the box.

There's a couple of plugins that have caught my eye for that I've been thinking of testing out:

https://github.com/aguingand/tiptap-markdown
